webapp/src/components/jira_ticket_tooltip/index.ts

```diff
@@ -0,0 +1,24 @@
+import {connect} from 'react-redux';
+import {bindActionCreators, Dispatch} from 'redux';
+import {GlobalState} from 'mattermost-redux/types/store';
+
+import {isUserConnected, getIssue, getUserConnectedInstances, getDefaultUserInstanceID} from 'selectors';
+import {getIssueByKey} from 'actions';
+
+import TicketPopover from './jira_ticket_tooltip';
+
+const mapStateToProps = (state: GlobalState) => {
+    return {
+        connected: isUserConnected(state),
+        ticketDetails: getIssue(state).ticketDetails,
+        isLoaded: getIssue(state).isLoaded,
```

Comment:
What happens when we hover over one Jira issue link, and then hover over a different one?

Reply:
We get the tooltip of the second link in that case, which was already fetched when the link was posted in the channel.